src/fits_schema/output_template.py

```diff
@@ -94,7 +94,7 @@ def column_template(column: Column) -> Generator[str]:
     if not hasattr(column, "tform_code"):
         raise AttributeError(f"Missing tform_code for column {column}")
 
-    yield f"TFORM# = {column.tform_code:20s} / {column.dtype.__name__}"
```

Comment:
You could just use the str repr of dtype?

Reply:
That gives e.g. `<class 'numpy.float64'>` instead of `Float64`.  I think exposing the fact that numpy is used underneath is maybe not so nice.  It's meant to just be human-readable.

Reply:
After this fix, it looks like:
```
TTYPE# = EVENT_ID             / ID of this event, unique within an observation
TFORM# = K                    / Int64

TTYPE# = TIME                 / Event time of arrival, as an MET
TFORM# = D                    / Double
TUNIT# = s                    / or convertible to 'time'
TUCD#  = time                
```

Which I think is fine.  It's only a comment, and not required for the template, it's more just to help a reader remember what "K"  or "D" means